packages/features/auth/lib/next-auth-options.ts

```diff
@@ -950,7 +950,7 @@ export const getOptions = ({
               return true;
             }
           } else if (existingUserWithEmail.identityProvider === IdentityProvider.CAL) {
-            return "/auth/error?error=use-password-login";
+            return `/auth/error?error=use-identity-login&provider=${existingUserWithEmail.identityProvider}`;
```

Comment:
I think the error code `use-identity-login` doesn't make sense in this case as it is email and password case and not identity provider.

How about we change the error to something generic like, "wrong-provider" and then we can handle all cases under this error instead of use-identity-login